web/server/vue-cli/src/views/Reports.vue

```diff
@@ -115,7 +115,7 @@
             }}"
             class="file-name"
           >
-            {{ item.checkedFile }}
+            {{ item.checkedFile }} @ Line {{ item.line }}
```

Comment:
```suggestion
            {{ item.checkedFile }} @&nbsp;Line&nbsp;{{ item.line }}
```
Otherwise the line is broken at weird places. By the way what is the motivation behind this change? We should consider a more concise format:
```suggestion
            {{ item.checkedFile }}:{{ item.line }}
```

Reply:
Good point to not to brake the line at weird places. By the way this is how it looked like in the old UI.